README.md

```diff
@@ -25,7 +25,7 @@
 
 # 📖 [MEME-GALAXY] <a name="about-project"></a>
 
-**Meme Galaxy** is a single-page app built using React JS and Vite whose aim is to display different memes with differenct categories accodrding to data fetched from <b>imgflip API.</b> Further information is available in the <a href="https://drive.google.com/file/d/1eZNRTECZUpxfykamc0erwobgSFsP5tY5/view?usp=sharing">video presentation</a>
+**Meme Galaxy** is a single-page app built using React JS and Vite whose aim is to display different memes with differenct categories accodrding to data fetched from <b>imgflip API.</b> For more information visit the <a href="https://drive.google.com/file/d/1eZNRTECZUpxfykamc0erwobgSFsP5tY5/view?usp=sharing">video presentation</a>
 ## 🛠 Built With <a name="built-with"> React JS</a>
 
```

Comment:
- [ ] In your project, I can only see snapshot tests. But in this milestone. Students write unit and integration tests as well. Kindly implement them. [Is the snapshot test is a unit test?](https://www.google.com/search?q=is+snapshot+test+is+a+unit+test&rlz=1C5CHFA_enUZ1037UZ1037&sxsrf=APwXEdfPLsf66rQX34QRv-M_y6JaYZzwrg%3A1685123767662&ei=t_JwZLSBKLWMxc8PuviKuAg&ved=0ahUKEwj0uMucx5P_AhU1RvEDHTq8AocQ4dUDCBA&uact=5&oq=is+snapshot+test+is+a+unit+test&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQAzIECCMQJzoKCAAQRxDWBBCwAzoICCEQoAEQwwRKBAhBGABQ_gJYiwhg6wloAnABeACAAcUBiAGmBZIBAzAuNJgBAKABAcABAcgBCA&sclient=gws-wiz-serp)

> Remember:
> The app has unit tests for several components of the app and some pure functions in Redux code.
> The app has integration tests using React Testing Library. You can perform a mocking data fetching and Redux Store connection.